packages/cli/templates/backend-plugin/src/plugin.ts.hbs

```diff
@@ -3,7 +3,7 @@ import {
   createBackendPlugin,
 } from '@backstage/backend-plugin-api';
 import { createRouter } from './router';
-import { catalogServiceRef } from '@backstage/plugin-catalog-node';
```

Comment:
I'm not sure we want to make this change? @freben?

I think that these changes are basically going backwards, and we want to be using the service everwhere from the root export.

Reply:
But these changes are not in the scaffolder package. They're part of the cli templates, we should not be changing any files that aren't in the scaffolder packages in this PR i think :pray: